src/beamlime/handlers/accumulators.py

```diff
@@ -118,8 +145,16 @@ def add(self, timestamp: int, data: DetectorEvents) -> None:
     def get(self) -> sc.DataArray:
         # Could optimize the concatenate by reusing a buffer (directly write to it in
         # self.add).
-        pixel_ids = np.concatenate([c.pixel_id for c in self._chunks])
-        time_of_arrival = np.concatenate([c.time_of_arrival for c in self._chunks])
+        pixel_ids = (
+            np.concatenate([c.pixel_id for c in self._chunks])
+            if self._chunks
+            else np.array([], dtype=np.int32)
+        )
+        time_of_arrival = (
+            np.concatenate([c.time_of_arrival for c in self._chunks])
+            if self._chunks
+            else np.array([], dtype=np.int32)
+        )
```

Comment:
Unrelated to main change, but added tests showed that Numpy does not like to concat length 0.